auth/src/main/res/layout/fui_check_email_layout.xml

```diff
@@ -1,38 +1,57 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <LinearLayout
-        style="@style/FirebaseUI.WrapperStyle"
-        android:layout_height="wrap_content"
-        android:orientation="vertical">
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
 
-        <android.support.design.widget.TextInputLayout
-            android:id="@+id/email_layout"
-            style="@style/FirebaseUI.TextInputLayout.EmailField"
-            android:layout_width="match_parent"
+        <LinearLayout
+            style="@style/FirebaseUI.WrapperStyle"
             android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/fui_field_padding_vert"
-            android:transitionGroup="true"
-            android:transitionName="@string/fui_email_field_name"
-            app:errorEnabled="true"
-            tools:ignore="UnusedAttribute">
+            android:orientation="vertical">
 
-            <android.support.design.widget.TextInputEditText
-                android:id="@+id/email"
-                style="@style/FirebaseUI.TextInputEditText.EmailField" />
+            <android.support.design.widget.TextInputLayout
+                android:id="@+id/email_layout"
+                style="@style/FirebaseUI.TextInputLayout.EmailField"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/fui_field_padding_vert"
+                android:transitionGroup="true"
+                android:transitionName="@string/fui_email_field_name"
+                app:errorEnabled="true"
+                tools:ignore="UnusedAttribute">
 
-        </android.support.design.widget.TextInputLayout>
+                <android.support.design.widget.TextInputEditText
+                    android:id="@+id/email"
+                    style="@style/FirebaseUI.TextInputEditText.EmailField" />
 
-        <Button
-            android:id="@+id/button_next"
-            style="@style/FirebaseUI.Button"
-            android:text="@string/fui_next_default" />
+            </android.support.design.widget.TextInputLayout>
 
-    </LinearLayout>
+            <Button
+                android:id="@+id/button_next"
+                style="@style/FirebaseUI.Button"
+                android:text="@string/fui_next_default" />
 
-</ScrollView>
+            <TextView
+                android:id="@+id/email_tos_and_pp_text"
+                style="@style/FirebaseUI.Text.BodyText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/fui_field_padding_vert"
+                android:textIsSelectable="false"
+                tools:text="Terms and Privacy Policy" />
+
+            <TextView
```

Comment:
What's the difference between this TextView and the one above? If this isn't a mistake, we should add a comment explaining that.

Reply:
Done.